lib/stm32/f0/rcc.c

```diff
@@ -43,6 +43,7 @@
 /* Set the default clock frequencies */
 uint32_t rcc_ahb_frequency = 8000000; /* 8MHz after reset */
 uint32_t rcc_apb1_frequency = 8000000; /* 8MHz after reset */
+uint32_t rcc_lse_frequency = 32768;		/* most common default */
```

Comment:
I've gone back and forth on that. Do I put it into the lse_enable() ? Does anyone use an LSE that **isn't** 32768Hz ? It is needed if we are computing the baud rate than the baud rate clock is connected to the LSE, but I didn't feel comfortable just _assuming_ it was going to be 32768.